tests/test_telemetry.py

```diff
@@ -371,6 +370,62 @@ def test_app_dependencies_loaded(self):
             if not seen:
                 raise Exception(dependency + " not recieved in app-dependencies-loaded message")
 
+    def setup_app_dependency_loaded_not_sent_dependency_collection_disabled(self):
+        weblog.get("/enable_product")
+
+    @missing_feature(
+        context.library in ("dotnet", "nodejs", "java", "golang", "python"),
+        reason="Weblog endpoint GET /enable_product is not implemented yet. ",
+    )
```

Comment:
I believe golang is the only language that we should be skipping here and the reason should say that telemetry is missing instead of the endpoint not being implemented